pkg/controllers/provisioning/provisioner.go

```diff
@@ -234,6 +234,10 @@ func (p *Provisioner) NewScheduler(ctx context.Context, pods []*v1.Pod, stateNod
 		if err != nil {
 			return nil, fmt.Errorf("getting instance types, %w", err)
 		}
+		if len(instanceTypeOptions) == 0 {
+			logging.FromContext(ctx).With("provisioner", nodePool.Name).Info("skipping provisioner, no resolved instance types found")
```

Comment:
```suggestion
			logging.FromContext(ctx).With(lo.Ternary(nodePool.IsProvisioner, "provisioner", "nodepool"), nodePool.Name).Info("skipping, no resolved instance types found")
```

We need to handle the logging for both cases when we support NodePools